script/integration/containerd/entrypoint.sh

```diff
@@ -23,8 +23,8 @@ set -euo pipefail
 source "/utils.sh"
 
 PLUGIN=stargz
-REGISTRY_HOST=registry-integration
-REGISTRY_ALT_HOST=registry-alt
+REGISTRY_HOST=registry-integration.io
+REGISTRY_ALT_HOST=registry-alt.io
```

Comment:
I guess we need to fix nerdctl to accept the old hostname.

Changing the hostname is ok as a workaround, but I guess we should use .local rather than .io